lib/meson.build

```diff
@@ -0,0 +1,70 @@
+# Compiled library
```

Comment:
I don't really have any comments about this file in particular, but I don't think `/lib` is the right place for the shared library implementation files; `/src` would be my preference. `/lib` to me suggests that I'd find static libraries or other build output in that directory.

Reply:
Ok, I chose it because I have a project that contains both library and an executable, so i put library headers in `include/`, library sources in `lib/` and sources of the executable in `src/`. But here it doesn't make much sense :)